### ISSUE TITLE
Migrate ATD to latest version

### ISSUE THREAD

**Issue by saikrishna321** [znsio/teswiz#130]
Hi Folks, 

Thanks for using ATD. We have made quite a few changes to the ATD core which now depends on `appium2.0`. Upcoming release will be have breaking changes. 

Steps to follow: 

1. Update the dependency as below, currently in jitpack.
```
	<dependency>
	    <groupId>com.github.AppiumTestDistribution</groupId>
	    <artifactId>AppiumTestDistribution</artifactId>
	    <version>appium2.0-6de74eab6c-1</version>
	</dependency>
```

2. Make sure you have Appium2.0 server installed with respective drivers. 
3. Install `appium-device-farm` plugin. Refer https://github.com/AppiumTestDistribution/appium-device-farm
4. Update your capabilties.json as https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/appium2.0/caps/iOScapabilities.json
5. If you want to run your test against BrowserStack Refer: https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/appium2.0/caps/browserstack.json
6. PCloudy Capabilties.json Refer: https://github.com/AppiumTestDistribution/AppiumTestDistribution/blob/appium2.0/caps/pcloudy_capabilities.json

cc: @anandbagmar

**Comment by saikrishna321** [comment]
Any progress on this migration? I would like to add this repo as reference for cucumber runner under ATD.

**Comment by anandbagmar** [comment]
Latest ATD is now merged with teswiz main (starting from teswiz v0.0.80)
